### PR TITLE
Scope changes

### LINKED ISSUE #14: Surfacing "Technical improvement planned compared to the draft state" better
The Deliverable section includes a list of documents, expected deadlines, current drafts, and other information that is be mostly about the mechanics of publishing things.

For some of these documents, it also includes "Technical improvement planned compared to the draft state". This is important information, and its relatively easy to miss it where it currently is. Consolidating this in a goals section outside of the deliverable section would help explain the purpose and goals of the Working Group. This may offer part of the solution to #13 

Among these is "Allowing the HTML Serialization of HTML5". I think this is a major change to EPUB, that is well worth highlighting: both because the benefits of doing so are considerable, but so are the compatibility implications. This may end up being the single most impactful thing this WG does, or it may be the reason why some will oppose it's creation. This should be much earlier and much more prominent in the charter.

### LINKED ISSUE #16: Distinguish EPUB 3.2 from EPUB 3.X
In the scope it seems a little unclear that EPUB 3.2 and EPUB 3.X are different.
I hope it is clearly stated that EPUB 3.2 spec will be kept as is, and EPUB 3.X will be the following version.

### LINKED ISSUE #17: Change "Allowing the HTML Serialization of HTML5" to "Considering allowing ~"
About "Allowing the HTML Serialization of HTML5" stated below,
 &gt; 2.1 Normative Specifications
  &gt;EPUB Content Documents 3.X
    &gt;Technical improvement planned compared to the draft state:
      &gt; Improved Alignment with Web Platform:

I am afraid of "allowing" becoming a premise of the charter. HTML Serialization needs a lot of argument because it has a big impact on reading system and check tools.
So I propose changing the phrase to "Considering allowing the HTML Serialization of HTML5"

### PR DESCRIPTION
**Pull request by iherman**
- The list of "Technical improvement planned compared to the draft state" for the deliverables have been removed, and the content moved to the scope section. (Issue #14)
    - an overall remark has been added drawing attention on the possible effects on deployed systems and checking tools. This covers, in particular, the HTML feature. (Issues #14 and #17)
- Added an extra statement "defining" EPUB 3.X, also making clear that EPUB 3.2 will not change. (Issue #16)
- Added a short statement (mostly inspired by the [separate blog](https://www.w3.org/blog/2020/03/listen-to-the-people-the-future-of-epub-and-new-directions-for-publishing-w3c/)) on the main goal of the work. (At least a partial answer to #13)
- Taken over the language from https://github.com/w3c/epub-3-wg-charter/issues/12#issuecomment-613103960 on `epub:type` (Issue #12)
- (Handled an HTML Markup error)

Preview: https://raw.githack.com/w3c/epub-3-wg-charter/scope-changes/index.html  
Diff: https://tinyurl.com/qrvzh8v

- Fix #14
- Fix #17
- Fix #16

@frivoal, you may realize that I did not add a 'fix' for issue #13, because I am not sure this PR fully answers your concerns.

@mattgarrish, @dauwhe, I have not added #12 as a 'fix', because I am not sure this is the whole picture for that issue...

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -213,15 +221,24 @@ <h2>Scope</h2>
 						Enhancements and improvements to internationalization of technologies defined for EPUB to ensure global usability. While the content of a digital publication (HTML, CSS, etc.) already have advanced internationalization features, an EPUB 3 publication also includes additional textual content like metadata, table of contents, etc., which must abide to all requirements of internationalization. 
 					</li>
 					<li>
-						Integration of new Open Web Platform technologies necessary for enhanced digital publishing experiences, including HTML5 for content documents and new core media types for content.
+						Continued integration of errata and bug fixes.
 					</li>
 					<li>
-						Continued integration of errata and bug fixes.
+						Integration of new Open Web Platform technologies for enhanced digital publishing experiences; these may include:
```
Comment (mattgarrish) [inline index.html]: To one of Florian's points I saw, should this perhaps be more open-ended, so maybe changed to "these may include but are not limited to:" ?

**Review by mattgarrish — APPROVED** [review]
Looks fine as far as my comment about epub:type goes. I don't know the answer to the SSV question and we probably need to ponder that some more. My only driving concern is whatever status it gets matches what we need to get the specs to REC.

File: index.html (hunk 1 above)
Comment (iherman) [reply index.html]: Right. Added it in https://github.com/w3c/epub-3-wg-charter/pull/18/commits/90164e3f06e244d9f54bee76b4bddc12270f839a

File: index.html [hunk 2]
```diff
@@ -192,7 +192,15 @@ <h1>DRAFT</h1>
...
+				</p>
+					
+				<p>
+					However, the interoperability of EPUB 3 content, and the conformance of EPUB 3 reading systems, remain a serious issue. There is a need for a more rigorous and comprehensive testing of EPUB 3; this may also mean the need for a further clarification of the standard text itself. (See also the <a href="https://www.w3.org/blog/2020/03/listen-to-the-people-the-future-of-epub-and-new-directions-for-publishing-w3c/">separate blog</a> for more background.) The EPUB 3 Working Group will advance, refine, and clarify the EPUB 3 standard to ensure its relevance as digital publishing continues to evolve, and specify a comprehensive testing environment for EPUB 3. 
+				</p>
+
+				<p>
+					The Working Group will produce a new version of EPUB 3, codenamed “EPUB 3.X” for the purposes of this charter; the Working Group will have to define the correct numbering scheme (e.g., EPUB 3.2.1, or EPUB 3.3, etc.). The Working Group will <em>not</em> change the EPUB 3.2 version itself, which will therefore remain unchanged.
```
Comment (dauwhe) [inline index.html]: Perhaps the last sentence could be 

> The Working Group will <em>not</em> change EPUB 3.2 itself.

Comment (iherman) [reply index.html]: Done.

**Comment by murata2makoto** [comment]
Why does this PR mentions SSML?  It is heavily used by some textbook publishers in Japan.  Please don't touch it.

**Comment by iherman** [comment]
@murata2makoto the context is 

> allowing the HTML Serialization of HTML5, including whether and how to handle xml-specific technologies like epub:type and the ssml:* attributes; 

Ie, _if_ HTML serialization is used, we have to look what it means for ssml. Maybe the answer is: if you as an author want to use ssml, you _must_ use XHTML. And that would be perfectly fine.

Ie, it does not say we would touch ssml itself.

**Review by wareid — APPROVED** [review]

**Comment by iherman** [comment]
See https://github.com/w3c/epub-3-wg-charter/issues/17#issuecomment-613763909 and https://github.com/w3c/epub-3-wg-charter/issues/16#issuecomment-613763844 on the parts relevant to #16 and #17.

**Comment by frivoal** [comment]
Thanks. I think it is a good PR, and support merging it.

As for whether it addresses #13, I think it mostly does, and I'm OK with it being closed. It makes it much easier to figure out what the group may do, and gives a reasonably clear motivation for the compatibility work, as well as the globalization work. For the "Integration of new Open Web Platform technologies" part, the motivation/intended audience is less clearly stated, but at least it's upfront about it, and the subsequent sentence makes it clear that these are ideas to be considered and that impact on compatibility must be taken into account, so things should be reasonably clear to all would-be participants, regardless of how they feel about such things.

**Comment by iherman** [comment]
Thanks @frivoal.

Per all the received comments, merging this PR, and will also close #13. @mattgarrish and @dauwhe, I will leave #12 open unless you believe it is fine to close that one, too.

**Comment by mattgarrish** [comment]
> I will leave #12 open unless you believe it is fine to close that one, too

Might be good to close it and open a separate issue about the SSV, since that spun out of the discussion.

**Pull request merged**